<launch>
<node name="ros_snu_emulator" pkg="ros_swm_gps" type="ros_snu_emulator" output="screen" />
</launch>
